terraform/provider.tf

```
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
```

As this is a terraform module intended to be used from within a larger terraform project, the provider versions should only be pinned using `>=` constraints to allow for maximum flexibility in the larger project. Otherwise if CIRRUS-ASF needs to upgrade the provider version you also need to make a PR to this repo to upgrade it here.

https://developer.hashicorp.com/terraform/language/expressions/version-constraints#terraform-core-and-provider-versions

```suggestion
      version = ">= 3.0, != 3.14.0"
```

---

The `!= 3.14.0` comes from older versions of cumulus that had that for some reason. I'm actually not sure what was wrong with that particular release of the provider, might be best to drop that constraint actually, although practically it should never matter.